styles/onyx/components/table/_core.scss

```diff
@@ -165,6 +165,17 @@
       }
     }
   }
+
+  th,
```

Comment:
This table happened to need this, but I feel it will be useful for any kind of table, so I put it here.